core/modules/hubot/responder.js

```diff
@@ -35,24 +35,24 @@ Responder.prototype.random = function (arr) {
 
 Responder.prototype.send = function () {
     for (var i = 0; i < arguments.length; i++) {
-		switch(getType(arguments[i])) {
-			case 'message': {
-				this.api.sendMessage(arguments[i], this.event.thread_id);
-				break;
-			}
-			case 'url': {
-				this.api.sendUrl(arguments[i], this.event.thread_id);
-				break;
-			}
-			case 'image': {
-				this.api.sendImage('url', arguments[i], '', this.event.thread_id);
-				break;
-			}
-			case 'file': {
-				this.api.sendFile('file', arguments[i], '', this.event.thread_id);
-				break;
-			}
-		}
+        switch (getType(arguments[i])) {
+            case 'message': {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 16 space characters but found 24.](https://www.codacy.com/app/theprojectshop/Kassy/file/2589168506/issues/source?bid=3282987&fileBranchId=3291254#l39)